app/src/main/java/org/p2p/wallet/bridge/send/ui/BridgeSendFragment.kt

```diff
@@ -195,7 +195,7 @@ class BridgeSendFragment :
         binding.widgetSendDetails.showDelayedFeeViewLoading(isLoading)
     }
 
-    override fun showFeeViewVisible(isVisible: Boolean) {
+    override fun showFeeVisible(isVisible: Boolean) {
```

Comment:
Maybe would better:
```suggestion
    override fun setFeeIsVisible(isVisible: Boolean) {
```

Reply:
Sometimes people do say missing `is`, or `am` and so on. 
Therefore, I would say it doesn't make any sense. I am trying always to make the names as shorter as possible